src/index.html

```diff
@@ -7,6 +7,239 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="head">
+      <img class="head__moyo"
+      src="images/logo.png" alt="moyo">
+      <nav class="tags">
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link">
            ....
```
Найчастіще ти будеш зустрічатись з такою розміткою навігації